### DISCUSSION TITLE
BT-99 in efx-templates

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#998] [Ideas]
We are of the opinion that BT-99 (Information about review deadlines) would be better placed in ‘5.1.16 Further information, mediation and review’ in the efx-templates.

What is the subdivision in the efx templates based on?
Is there any chance that this will be changed?

**Comment by pdonohoe** [comment]
Thank you for your comment. After discussion, we agree with your suggestion. BT-99 will be displayed below the organisation "Review organisation" in section "5.1.16 Further information, mediation and review" on the release of SDK-1.13. This will be applied to previous versions of the SDK soon afterwards.

**Comment by JanEmde** [comment]
Thank you very much. Looking forward to it :)
